test/Db2FormatterTest.js

```diff
@@ -20,6 +20,22 @@ describe("Db2Formatter", function() {
         );
     });
 
+    it("formats only -- as a line comment", function() {
+        const result = sqlFormatter.format(
+            "SELECT col#1 FROM\n" +
+            "-- This is a comment\n" +
+            "MyTable;\n",
+            {language: "db2"}
+        );
+        expect(result).toBe(
+            "SELECT\n" +
+            "  col#1\n" +
+            "FROM\n" +
+            "  -- This is a comment\n" +
+            "  MyTable;\n"
+        );
+    });
```

Comment:
We should also test the formatting of `@` and `#` inside identifiers. This test partially covers the latter, but it would be better to have a separate test with clear description, explaining the `@` and `#` are part of identifiers (e.g. not treated as operators).